pages/sendtoken.php

```diff
@@ -31,11 +31,26 @@
 $ldap = "";
 $userdn = "";
 $token = "";
+$usermail = "";
 
 if (!$mail_address_use_ldap) {
-    if (isset($_POST["mail"]) and $_POST["mail"]) { $mail = strval($_POST["mail"]); }
-     else { $result = "mailrequired"; }
-}
+    if (isset($_POST["mail"]) and $_POST["mail"]) {
+        $mail = $_POST["mail"];
```

Comment:
Can you use `strval(` for consistency please.

Reply:
Done in #411